data/categories.txt

```diff
@@ -3099,6 +3136,156 @@ sk:4Pomník|pamiatka|atrakcia|zaujímavosť
 sw:Makumbusho|kumbukumbu ya kihistoria|kivutio|utalii
 fa:یادبود تاریخی
 
+historic-memorial-cross
+en:Memorial cross
+ar:الصليب التذكاري
+be:Памятны крыж
+bg:Мемориален кръст
+cs:Pamětní kříž
+da:Mindesmærkekors
+de:Gedenkkreuz
+el:Σταυρός μνήμης
+es:Cruz conmemorativa
+eu:Oroimenezko gurutzea
+fa:صلیب یادبود
+fi:Muistoristi
+fr:Croix commémorative
+he:צלב זיכרון
+hu:Emlékkereszt
+id:Salib peringatan
+it:Croce commemorativa
+ja:メモリアルクロス
+ko:기념 십자가
+mr:मेमोरियल क्रॉस
+nb:Minnekors
+nl:Herdenkingskruis
+pl:Krzyż pamiątkowy
+pt:Cruz memorial
+pt-BR:Cruz de memorial
+ro:Cruce comemorativă
+ru:Памятный крест
+sk:Pamätný kríž
+sv:Kors till minne
+sw:Msalaba wa kumbukumbu
+th:อนุสรณ์สถานข้าม
+tr:Anıt haç
+uk:Пам'ятний хрест
+vi:Thánh giá tưởng niệm
+zh-Hans:纪念十字架
+zh-Hant:紀念十字架
+
+historic-memorial-plaque
+en:Commemorative plaque
+ar:لوحة تذكارية
+be:Памятная дошка
+bg:Паметна плоча
+cs:Pomník
+da:Mindesmærke
+de:3Gedenktafel
+el:Μνημείο
+es:Placa conmemorativa
+eu:Oroitzapenezko plaka
+fa:گردشگری
+fi:Muistomerkki
+fr:Plaque commémorative
+he:לוחית זיכרון
+hu:Emlékmű
+id:Tugu peringatan
+it:Memoriale
+ja:銘板
+ko:기념비
+mr:फलक
+nb:Minnesmerke
+nl:Plaquette
+pl:Tablica pamiątkowa
+pt:Placa comemorativa
+pt-BR:Placa comemorativa
+ro:Comemorare
+ru:Памятная доска
+sk:Pomník
+sv:Minnesmärke
+sw:Plaque
+th:ที่ระลึก
+tr:Hatıra Plaketi
+uk:Дошка пам'яті
+vi:Đài tưởng niệm
+zh-Hans:纪念牌匾
+zh-Hant:紀念牌匾
+
+historic-memorial-stolperstein
+en:Stolperstein
+ar:حجر عثرة
+be:Камяні спатыкнення
+bg:Cпънка
+cs:Stolperstein
+da:Stolpersten
+de:3Stolperstein
+el:Λίθοι Μνήμης
+es:Stolperstein
+eu:Stolperstein
+fa:سنگ مانع
+fi:Kompastuskivet
+fr:Stolperstein
+he:אבן נגף
+hu:Botlatókő
+id:Stolperstein
+it:Pietre d'inciampo
+ja:ストルパーシュタイン
+ko:장애물
+mr:अडथळा
+nb:Snublesteiner
+nl:Stolperstein
+pl:Stolperstein
+pt:Stolperstein
+ro:Blocaj de poticnire
+ru:Камни преткновения
+sk:Kameň úrazu
+sv:Snubbelsten
+sw:Kikwazo
+th:อุปสรรค์
+tr:Stolperstein
```

Comment:
Tökezleyen Taşlar

Don't forget to correct this too

Reply:
Oh, I forgot, yes ..